File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break


for i in range(n):
    print("")
    for j in range(n - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")


print("")

File: mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int n = get_int("Size: \n");
    //rows
    for (int i = 0; i < n; i++ )
    {
        //pushing the left side
        for( int k = 0; k < n  - 1- i; k++)
        {
            printf(" ");
        }
        //printing the left side
        for (int j = 0; j < i + 1; j++)
        {

            printf("#");

        }
        //space between
        printf(" ");

        //right side
        for (int l = 0; l < i + 1; l++)
        {

            printf("#");

        }

        //prints new line
        printf("\n");
    }
}